Update online_archives.md (#13225)
DataDog/documentation: content/en/logs/log_configuration/online_archives.md

## Searching in Online Archives

> Select Online Archives from the dropdown in the Logs Explorer to begin searching in Online Archives instead of indexes. Find this dropdown next to the time picker. You can adjust the time picker with pre-set options, such as 2 or 3 months, or by selecting the calendar view to search further back in time.

Author: ```suggestion
Select Online Archives from the dropdown in the Logs Explorer to begin searching in Online Archives instead of indexes. Find this dropdown next to the time picker. You can adjust the time picker by selecting pre-set options, up to 3 months, or by selecting the calendar view to search further back in time.
```

{{< img src="logs/log_configuration/online_archives/searching.png" alt="How to search your online archive" style="width:100%;">}}

[Search][4] by typing in queries in the search bar or by selecting the relevant facet in the facet panel.

**Notes**: 

> - You cannot export online archive logs to dashboards, notebooks, or monitors.

Author: ```suggestion
- You cannot export online archive logs to Dashboards, Notebooks, or Monitors.
```

> - The transactions and patterns view is not available for Online Archives.

Author: ```suggestion
- The Transactions and Patterns view is not available for Online Archives.
```
